test/representers/api/story_representer_test.rb

```diff
@@ -64,15 +64,6 @@
       json['_links']['profile']['href'].must_equal 'http://meta.prx.org/model/story'
     end
 
-    it 'has a profile for the default image' do
```

Comment:
Is it no longer "usual" to include a profile link on the image links?  (Looks like you've standardized them here to include `href` and `title`, but not `profile`).